docs/docs/references/users/README.md

```diff
@@ -80,10 +80,137 @@ A user that signed in with Facebook:
 
 You can query the user profile using
 [Admin Console](../../../docs/recipes/manage-users/using-admin-console#view-and-update-user-profile) or
-[Management API](/docs/recipes/manage-users/using-management-api), such as <a href="/api/#tag/Users/paths/~1api~1users~1:userId/get" target="_blank">`GET /api/users/:userId`</a>.
+[Management API](../../../docs/recipes/manage-users/using-management-api),
+such as <a href="/api/#tag/Users/paths/~1api~1users~1:userId/get" target="_blank">`GET /api/users/:userId`</a>.
 
 ### Basic data
 
+_Basic data_ contains the basic user info in pre-defined attributes.
+
+#### User ID
+
+_User ID_ is a unique auto-generated key to identify the user in Logto.
+
+#### Username
+
+_Username_ is used for sign-in with _username_ and password.
+
+When the user has not registered with their username, their _username_ may be empty.
+When the _username_ is non-empty, it should be at most 128 characters, only contain letters, numbers, and underscores (`_`), and NOT start with a number.
+
+#### Primary email
+
+_Primary email_ is the user's email address, used for sign-in with the email and passcode.
+
+When the user has not registered with their email, their _primary email_ may be empty.
+When the _primary email_ is non-empty, it should be at most 128 characters.
+
+#### Primary phone
+
+_Primary phone_ is the user's phone number, used for sign-in with the phone number and passcode from SMS.
+
+When the user has not registered with their phone number, their _primary phone_ may be empty.
+When the _primary phone_ is non-empty, it should contain numbers prefixed with the
+[country calling code](https://en.wikipedia.org/wiki/List_of_country_calling_codes) (excluding the plus sign `+`).
+
+#### Name
+
+_Name_ is the user's full name.
+It should be at most 128 characters.
+
+#### Avatar
+
+_Avatar_ is the URL pointing to the user's avatar image.
+
+If the user registers with a social connector like Facebook and WeChat, their _avatar_ may be retrieved from the social user info.
+It should be at most 2048 characters.
```

Comment:
nit: How about saying "the max-length is xxx" instead of "it should be"

Reply:
Updated to 
```suggestion
Its max length is 2048.
```